bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/Variant.java

```diff
@@ -703,7 +703,7 @@ private void updateObsFlags() {
 			// can not be scheduled at all tonight.
 			Union<Interval> darkUnion = darkUnionCache.get(obs);
 			if (darkUnion == null) {
-				WorldCoords coords = obs.getCoords();
+				WorldCoords coords = obs.getCoords(getSchedule().getMiddlePoint());
```

Comment:
Same question here for the `BackgroundSolver`.  It seems like they should all work the same way.